examples/src/app/example-iframe.tsx

```diff
@@ -213,7 +213,7 @@ const ExampleIframe = (props: ExampleIframeProps) => {
         return false;
     };
 
-    const observer = new Observer({});
+    const observer = new Observer({}, {});
```

Comment:
Whoops, sorry but this is a mistake in the Observer JSDoc:

https://github.com/playcanvas/playcanvas-observer/blob/main/src/observer.js#L11

The options object should have square brackets around it to denote that it's optional.

Reply:
It's arguable that the first parameter should be marked as optional too, so you could be able to do:

```
const observer = new Observer();
```

@vkalpias what was the intention for the original API?

Reply:
Ahh yeah we should update the playcanvas-observer lib again in that case. I'll wait for @vkalpias answer.

Reply:
We should be able to make both arguments optional

Reply:
OK, I've attempted to fix this here: https://github.com/playcanvas/playcanvas-observer/pull/9